data/human/ships.txt

```diff
@@ -1260,6 +1260,63 @@ ship "Cruiser"
 
 
 
+ship "Cutthroat"
+	sprite "ship/pirate cutthroat"
+	thumbnail "thumbnail/pirate cutthroat"
+	attributes
+		category "Light Warship"
+		"cost" 1100000
+		"shields" 2800
+		"hull" 600
+		"required crew" 4
+		"bunks" 6
+		"mass" 171
+		"drag" 3
```

Comment:
```suggestion
		"drag" 3.5
```

The fact that this has 38% more engine space than the regular clipper should be more than enough of a speed boost without also reducing its drag by 22%. A tweak? Sure, seems appropriate. A 22% drop isn't a tweak, that's completely replacing the core structural members with something stronger. Reasonable on a low performing ship, but the Clipper's already pretty much top of its class as a high-performance ship. I doubt it has anything less than humanity's best already.

Reply:
Yet Marauders somehow find ways to improve even already top of the line ships like Falcons with little change to external appearance. Sure Southbound shipyard is a capable designer and manufacturer but without doubt there will be cuts so it's more economical to build just like every product today so there will be ways that things can be improved.

Reply:
I'm not saying that this can't have an improvement, I'm just positing that I don't think a 22% drop in drag is reasonable. The 3.5 I suggested *is* a improvement over the clipper's 3.7. So even if a pilot limited themselves to the same stock equipment as on a regular clipper, they'll still outpace it fairly easily. Throw in the fact that this has 138% of a clipper's engine capacity, and, well, there's zero question that this will be massively faster. This will be massively faster than a regular clipper even if the drag was left entirely alone.

Reply:
I'm afraid it wouldn't be enough to actually outrun most MELeviathan but turns out it still doesn't without reducing further to 2.6 or something. I'll bump it up to 3.4 and go nerf MELeviathan somehow.

Reply:
If the marauder Leviathan can outrun this Cuthroat, then the Leviathan needs to be nerfed. Likewise for any other marauder heavy warship. Top speed isn't everything, though. What's the acceleration look like? This thing has ~370 acceleration, which vastly outperforms every interceptor and even fighter, with the finch coming in at a lowly ~290 in comparison. (just looking at the in-game shipyards. I don't have the marauder ships edited into this game currently)

Reply:
Acceleration is proportional to thrust and mass, there's little mass increase with Leviathan so MELevi still have more acceleration than everything else.